aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import { Box } from 'grommet';
+import { ActivitiesNavigationalCards } from '../card';
+
+const skeleton = { message: { start: 'Loading', end: 'Content Loaded' } };
+
+export const DashBoardSkeletonExample = () => {
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => setLoading(!loading), 3000);
+  }, [loading]);
+
+  return (
+    <>
+      <Box width="large" skeleton={loading ? skeleton : undefined}>
```

Comment:
Should we use `Page` and `PageContent`? Does `skeleton` work on PageContent? (i'm thinking it should be PageContent is a Box.

Reply:
changed it to `page` and `pageContent` it jumps a little but let me know what you think